src/radical/pilot/pilot_description.py

```diff
@@ -8,6 +8,8 @@
 # ------------------------------------------------------------------------------
 # Attribute description keys
 #
+from radical.pilot import TaskDescription
+
```

Comment:
please use relative imports to avoid circular imports.

Reply:
ok

Reply:
Sorry, seems I wasn't clear.

Assume have this in `radical/pilot/__init__.py`:
```py
from .pilot_description import PilotDescription
```

and in `radical/pilot/pilot_description.py` we have something like
```py
import radical.pilot
```

then `import radical.pilot` will import `__init__.py` which will import `pilot_description.py` which will import `__init__.py` which will...  you get the picture.

To resolve that, please use this:
```py
from .task_description import TaskDescription
```

You will find that pattern throughout the code as well.

Reply:
got it. Cyclic dep